app/views/case_contacts/form/notes.html.erb

```diff
@@ -36,11 +36,7 @@
         <% end %>
 
         <%= button_tag( type: "submit" , class: "btn-sm main-btn primary-btn btn-hover" ) do %>
-          <% if step == @case_contact.form_steps.last %>
-            <i class="lni lni-checkmark-circle mr-5"></i> Submit
-          <% else %>
-            <i class="lni lni-checkmark-circle mr-5"></i> Save and continue
-          <% end %>
+          <%= render(Form::SubmitButtonComponent.new(last_step: @case_contact.form_steps.last, current_step: :notes)) %>
```

Comment:
If we are going to move the button into a component the component should encapsulate the whole button not just the contents of the button. That is the `button_tag` should be included in the component.